datastore/gcloud/aio/datastore/value.py

```diff
@@ -48,8 +48,8 @@ def from_repr(cls, data: Dict[str, Any]) -> 'Value':
         else:
             supported = [name.value for name in supported_types.values()]
             raise NotImplementedError(
-                f'{data.keys()} does not contain a supported value type'
-                f' (any of: {supported})')
+                '{} does not contain a supported value type'
+                ' (any of: {})'.format(data.keys(), supported))
```

Comment:
I find format strings to be a lot more readable when you use references instead of positional interpolation:
```python
'{vals} does not contain a supported value type'
' (any of: {supported})'.format(vals=data.keys(), supported=supported))
```

Reply:
Good call -- if we can't have f-strings, having references would be the next best thing.